File: CMSIS/DSP/cmsisdsp/sdf/scheduler/templates/cmsis.cpp
{
{% for ptr in ptrs %}
         {{theType}}* {{ptr}};
{% endfor %}
{% for ptr in inputs %}
         {{ptr[0]}}={{ptr[1]}}.getReadBuffer({{nb}});
{% endfor %}
{% for ptr in outputs %}
         {{ptr[0]}}={{ptr[1]}}.getWriteBuffer({{nb}});
{% endfor %}
         {{func}}({{args}},{{nb}});
         sdfError = 0;
       }

File: CMSIS/DSP/cmsisdsp/sdf/scheduler/templates/cmsis.py

{% for ptr in inputs %}
       {{ptr[0]}}={{ptr[1]}}.getReadBuffer({{nb}})
{% endfor %}
{% for ptr in outputs %}
       {{ptr[0]}}={{ptr[1]}}.getWriteBuffer({{nb}})
{% endfor %}
       {{outArgs}}[:]=dsp.{{func}}({{inArgs}})
       sdfError = 0
       